backend/schema/metadatadatabases.go

```diff
@@ -0,0 +1,55 @@
+package schema
+
+import (
+	"fmt"
+	"strings"
+
+	"google.golang.org/protobuf/proto"
+
+	schemapb "github.com/TBD54566975/ftl/protos/xyz/block/ftl/v1/schema"
+)
+
+type MetadataDatabases struct {
+	Pos Position `parser:"" protobuf:"1,optional"`
+
+	Calls []*Database `parser:"'database calls' @@ (',' @@)*" protobuf:"2"`
```

Comment:
This will need to be `'database' 'calls'`, because of the two-pass (tokenisation > parsing) system the Participle uses. That is, each individual token must be matched explicitly.